README.md

```diff
@@ -124,16 +124,19 @@ Animated GIFs will be displayed as still images to reduce loading time and the
 impact on performance of a client website.
 
 
-dev installation
+Dev installation
 ----------------
 
 ```
 git clone git@github.com:styladev/widget.git
 cd widget
-npm i
+npm install
 npm run build
+python -m SimpleHTTPServer
 ```
 
+then, find a demo page at `localhost:8080/dist/stage.html`
```

Comment:
seems to run on port :8000 on my machine, is it possible to pass the port as an option?